requirements.txt

```diff
@@ -1,4 +1,2 @@
-django==1.3
+django==1.9
```

Comment:
One other thing: we should be careful to use the same version of Django as Google App Engine.